array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,22 @@
+/* exported compact */
+// no false, no null, no Nan, no 0 or -0, no undefined,
+// no empty ----- maintain order
+
+function compact(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (!array[i]) {
+      continue;
```

Comment:
Generally speaking, using `continue` is a sign that you should reverse your logic. Rather than searching for values that are falsy, you should search for values that are truthy so you can do the intended work inside the conditional block here.